app/jobs/create_rss_notifications.rb

```diff
@@ -0,0 +1,17 @@
+class CreateRssNotifications < ApplicationJob
+  after_perform do |job|
+    self.class.set(:wait => 12.hours).perform_later(job.arguments.first)
+  end
+
+  def perform(options = {})
+    # Defaults for theforeman.org blog RSS
+    UINotifications::RssNotificationsChecker.new(options).deliver!
+    raise StandardError.new('OHNO')
```

Comment:
is this some leftover from testing?

Reply:
Definitely, it was meant to test what happens when notifications fail, my bad.